### DISCUSSION TITLE
Using copier without git?

### DISCUSSION THREAD

**Discussion by kefirbandi** [copier-org/copier#1634] [Ideas]
**Executive summary**
Currently copier won't work if git is not installed, even if you just want to use it to render and copy files from some local folder to another, where git is never involved in the process.

Would it be possible to remove git-dependence?

**Details**
Copier wants to find out whether the source directory has anything to do with git, and calls the git-executable to check this. But if it is not found then it fails.
My suggestion is to make a little change: when copier is unable to find git-executable then simply assume that source is not under git and proceed.
I tried this approach (modified `is_git_bundle` to return False if it fails to find git) and it works perfectly.

**Comment by yajo** [comment]
Yes, it should be possible to use Copier without git, unless the source and destination templates are git repos. This is a bug. Moved to issue tracker: https://github.com/copier-org/copier/issues/1696

FWIW you won't be able to use `copier update` unless using git.
